client/my-sites/site-settings/form-general.jsx

```diff
@@ -411,6 +396,31 @@ module.exports = React.createClass( {
 		);
 	},
 
+	renderTimezoneDropdown() {
+		if ( this.props.site.jetpack ) {
```

Comment:
We should create an issue or preemptively assume that this setting will be available in a future Jetpack release, circling back to add a `this.props.site.versionCompare` here.